src/components/custom/_agreement-label.js

```diff
@@ -51,12 +44,15 @@ const AgreementLabel = ({ handleChangeCheckbox, isChecked, color }) => {
                     fontSize="var(--text-size-xs)"
                     translate_text="I agree to the <0>terms and conditions</0>"
                     components={[
-                        <LinkText
+                        <LocalizedLinkText
+                            path="terms-and-conditions"
                             key={0}
-                            href={url}
-                            target="_blank"
+                            type="terms-and-conditions"
+                            external="true"
+                            to="terms-and-conditions"
                             size="14px"
                             color="red"
+                            target="_blank"
```

Comment:
This will not require `external` to be set to true as this will just be treated as another link on the page. We can remove `external`, `type`, `path` and `target`.